shim.js

```diff
@@ -5809,22 +5809,22 @@ dnode.prototype = {};
 function dnode (cons, opts) {
```

Comment:
Please make the changes to shim.coffee and then recompile to .js. The diff should be minimal.